### PR TITLE
Update getSignaturesForAddress and getConfirmedSignaturesForAddress2 RPC call description

### PR DESCRIPTION
**Pull request by jdcaballerov**
#### Problem
The description doesn't fully match the functionality of the call. Take for instance [this transaction](https://explorer.solana.com/tx/2ScmUjDCfyDgvDhdn2w2fHzZ4BaaQ3Vmx8fgvrR7DmcgTuM2gC9qp1F9ZBJiEUrY6YLHi68cDVfiu1ADdbWthKBn). The transfer in the transaction is never reported for the [mint involved](https://explorer.solana.com/address/6eAaYYYXexov1xSc9uizvgk6DY8eYVQiwxkXPvkVjGj4) . i.e the transaction never gets returned since the mint is not passed but the two ATAs thus misleading.

#### Summary of Changes

Modification suggestion

### CODE REVIEW

File: docs/src/developing/clients/jsonrpc-api.md [hunk 1]
```diff
@@ -1971,8 +1971,7 @@ Result:
 ### getSignaturesForAddress
 
 
-Returns confirmed signatures for transactions involving an
-address backwards in time from the provided signature or most recent confirmed block
+Returns confirmed signatures for transactions when the address is present in the compact-array of account addresses of a transaction. Returns signatures backwards in time from the provided signature or most recent confirmed block
```
Comment (CriesofCarrots) [inline docs/src/developing/clients/jsonrpc-api.md:1974]:
```suggestion
Returns signatures for confirmed transactions that include the given address in their `accountKeys` list. Returns signatures backwards in time from the provided signature or most recent confirmed block
```

**Review by CriesofCarrots — COMMENTED** [review]
Fair, makes sense to be more precise. Wdyt of my proposed wording?
Also, can you wrap to 80 chars?
Thank you!

File: docs/src/developing/clients/jsonrpc-api.md (hunk 1 above)
Comment (jdcaballerov) [reply docs/src/developing/clients/jsonrpc-api.md]: @CriesofCarrots done. LGTM!

File: docs/src/developing/clients/jsonrpc-api.md [hunk 2]
```diff
@@ -4367,8 +4368,8 @@ Result:
 **DEPRECATED: Please use [getSignaturesForAddress](jsonrpc-api.md#getsignaturesforaddress) instead**
 This method is expected to be removed in solana-core v2.0
 
-Returns confirmed signatures for transactions involving an
-address backwards in time from the provided signature or most recent confirmed block
+Returns confirmed signatures for transactions when the address is present in the compact-array of account addresses of a transaction. Returns signatures backwards in time from the provided signature or most recent confirmed block
+
```
Comment (CriesofCarrots) [inline docs/src/developing/clients/jsonrpc-api.md:4372]:
```suggestion
Returns signatures for confirmed transactions that include the given address in
their `accountKeys` list. Returns signatures backwards in time from the
provided signature or most recent confirmed block

```

**Comment by CriesofCarrots** [comment]
I just pushed a little commit making the two doc blocks the same. Thank you for helping improve our docs!

**Pull request merged**